### DISCUSSION TITLE
Essentials Discord Link

### DISCUSSION THREAD

**Discussion by Blzd87** [EssentialsX/Essentials#5398] [Ideas and Feature Suggestions]
please add essentials.link permission to your wiki permissions/commands page also add that requirement inside the discord link tutorial page as none of this is stated in the setup page.

**Comment by Evidentsinger14** [comment]
If you are referring to [This page](https://essinfo.xeya.me/permissions.html), it isn’t maintained by us officially, it is community maintained. May be better to make an issue on their issue tracker. 
as for the second part, what “requirement” are you referring to? Granting the permission?

**Comment by Blzd87** [comment]
https://essentialsx.net/wiki/Discord-Link.html

in page above... i think adding the fact you need to grant essentials.link perms to allow all users/players to use the /link command... as i followed the tutorial top to bottom and no one could use the link as they didnt have perms...

also noone who has linked is getting auto assigned a discord role
